packages/ui-module/pages/index.js

```diff
@@ -0,0 +1,14 @@
+import { createTheme } from '@platzily-ui/styling';
+import { ThemeProvider } from '@platzily-ui/styling';
+import StyledBox from '../components/style';
+
+export const theme = createTheme();
+
+
+export default function Home() {
+    return (    
+        <ThemeProvider theme={theme}>
```

Comment:
This should be in the _app.js file :grin:.

Reply:
I'm sorry😅, yes I will fix the problem.